x-pack/plugins/secops/public/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -132,15 +130,19 @@ export class AppKibanaFrameworkAdapter implements AppFrameworkAdapter {
       this.timezone = Private(this.timezoneProvider)();
       this.kbnVersion = kbnVersion;
       this.dateFormat = config.get('dateFormat');
+      try {
+        this.darkMode = config.get('theme:darkMode');
+      } catch (e) {
+        this.darkMode = false;
```

Comment:
When and why does this blow up? Do we have a standard way of logging this out as an error for us or an indepth user to look at? At least something so we know when this happens as a dev. Otherwise, later we won't know if this has been malfunctioning for a long time.

Reply:
They say that you can add a second parameter to the get with a default value. But I got an error, I will ask around.

Reply:
Thanks. Overall, I am seeing more and more programming languages such as GoLang and Rust not really using throws or exceptions as much if at all. I am just very cautious with them anytime I see them. Even if this just had a noisy console.log statement to let us know it's blowing up that would be fine with me.